src/tsdl_stubs.c

```diff
@@ -15,12 +15,17 @@
    that all foreign DLLs are already mapped into the process address space.
    Implicit linking is the simplest way to do that.
  */
-#include "SDL.h"
 void tsdl_nop (void) { SDL_WasInit(0); return; }
 #else
 void tsdl_nop (void) { return; }
 #endif
 
+/* Fixed arity function for OCaml, especially for macOS/clang. */
```

Comment:
Given the reference you gave shouldn't this rather be: `Fixed arity function, libffi does not support varargs`.